addons/event_sale/tests/test_event_internals.py

```diff
@@ -83,3 +84,72 @@ def test_event_registrable(self):
         })
         test_event_ticket.product_id.active = False
         self.assertEqual(test_event.event_registrations_open, False, 'cannot register if product linked to the tickets are all archived')
+
+
+class TestEventTicketData(TestEventSaleCommon):
+
+    def setUp(self):
+        super(TestEventTicketData, self).setUp()
+        self.ticket_date_patcher = patch('odoo.addons.event.models.event_ticket.fields.Date', wraps=FieldsDate)
+        self.ticket_date_patcher_mock = self.ticket_date_patcher.start()
+        self.ticket_date_patcher_mock.context_today.return_value = date(2020, 1, 31)
+
+    def tearDown(self):
+        super(TestEventTicketData, self).tearDown()
+        self.ticket_date_patcher.stop()
+
+    @users('user_eventmanager')
+    def test_event_ticket_fields(self):
```

Comment:
Seems like a lot of copy/paste from other test.
Maybe there is a way to factorize a bit?

Reply:
For future cleaning.